src/components/Navbar.js

```diff
@@ -1,18 +1,27 @@
+/* eslint-disable import/no-extraneous-dependencies */
```

Comment:
- [ ] Kindly remove all eslint-disabler from your code. 

> 🔧 Sometimes we may encounter challenges when using ESLint. Unexpected errors may arise which can be difficult to resolve quickly. If you come across an issue that is functioning properly but is still displaying an error message when uploaded to GitHub, I recommend disabling it in the .eslintrc.json file. This can help to prevent any confusion or delays in the development process. Let me know if you need any assistance.

Reply:
@Rustamxon7 kindly tell me how to do this  because am getting that error even when i install

Reply:
If you are not able to fix the issue. You can leave it and write why you were not able to solve it.